setup.py

```diff
@@ -2,19 +2,17 @@
 
 from setuptools import setup, find_packages
 
+# exclude i-pi-driver that is a compiled fortran file when installing from a local dir
+scripts = [str(p) for p in Path("bin").iterdir() if p.name != "i-pi-driver"]
 
 setup(
     packages=[
         *find_packages(exclude=["ipi_tests*"]),
         "ipi._driver",
         "ipi._driver.pes",
-        "ipi.tests",
-        "ipi.tests.regression_tests",
```

Comment:
So, I think people want to do some tests once they installed packages to know whether it is all working in their machine. Why take them all out?

Reply:
didn't work unless ran from a gh repo (in which case it picks up tests from the repo folder). never seen a package install tests in the python path.

Reply:
ping. shall we merge and then possibly open an issue to discuss whether we want to bundle tests, and in that case make sure they actually work?